src/data/thermal-conductivity-constants.ts

```diff
@@ -2050,6 +2050,730 @@ export const DIPPRThermalConductivityDictionary: Record<
+    thermalConductivityAtMaximumTemperature: 0.1068,
+  },
+
+  '1Hexyne': {
+    name: '1-Hexyne',
+    formula: 'C6H10',
+    CAS: '693-02-7',
+    molecularWeight: 82.1436,
+    C1: 0.21492,
+    C2: -0.0002899,
+    minimumTemperature: 141.25,
+    maximumTemperature: 344.48,
+    thermalConductivityAtMinimumTemperature: 0.174,
+    thermalConductivityAtMaximumTemperature: 0.1151,
+  },
+
+  '2Hexyne': {
+    name: '2-Hexyne',
+    formula: 'C6H10',
+    CAS: '764-35-2',
+    molecularWeight: 82.1436,
+    C1: 0.2119,
+    C2: -0.00027048,
+    minimumTemperature: 183.65,
+    maximumTemperature: 357.67,
+    thermalConductivityAtMinimumTemperature: 0.1622,
+    thermalConductivityAtMaximumTemperature: 0.1152,
+  },
+
+  Hydrazine: {
+    name: 'Hydrazine',
+    formula: 'H4N2',
+    CAS: '302-01-2',
+    molecularWeight: 32.04516,
+    C1: 1.3675,
+    C2: -0.0015895,
+    minimumTemperature: 274.69,
+    maximumTemperature: 623.15,
+    thermalConductivityAtMinimumTemperature: 0.9309,
+    thermalConductivityAtMaximumTemperature: 0.377,
+  },
+
+  Hydrogen: {
+    name: 'Hydrogen',
+    formula: 'H2',
+    CAS: '1333-74-0',
+    molecularWeight: 2.01588,
+    C1: -0.0917,
+    C2: 0.017678,
+    C3: -0.000382,
+    C4: -3.33e-6,
+    C5: 1.0266e-7,
+    minimumTemperature: 13.95,
+    maximumTemperature: 31.0,
+    thermalConductivityAtMinimumTemperature: 0.0754,
+    thermalConductivityAtMaximumTemperature: 0.0848,
+  },
+
+  HydrogenBromide: {
+    name: 'Hydrogen bromide',
+    formula: 'BrH',
+    CAS: '10035-10-6',
+    molecularWeight: 80.91194,
+    C1: 0.234,
+    C2: -0.0004636,
+    minimumTemperature: 185.15,
+    maximumTemperature: 290.62,
+    thermalConductivityAtMinimumTemperature: 0.1482,
+    thermalConductivityAtMaximumTemperature: 0.0993,
+  },
+
+  HydrogenChloride: {
+    name: 'Hydrogen chloride',
+    formula: 'ClH',
+    CAS: '7647-01-0',
+    molecularWeight: 36.46094,
+    C1: 0.8045,
+    C2: -0.002102,
+    minimumTemperature: 273.15,
+    maximumTemperature: 323.15,
+    thermalConductivityAtMinimumTemperature: 0.2303,
+    thermalConductivityAtMaximumTemperature: 0.1252,
+  },
+
+  HydrogenCyanide: {
+    name: 'Hydrogen cyanide',
+    formula: 'CHN',
+    CAS: '74-90-8',
+    molecularWeight: 27.02534,
+    C1: 0.43454,
+    C2: -0.0007008,
+    minimumTemperature: 259.83,
+    maximumTemperature: 298.85,
+    thermalConductivityAtMinimumTemperature: 0.2525,
+    thermalConductivityAtMaximumTemperature: 0.2251,
+  },
+
+  HydrogenFluoride: {
+    name: 'Hydrogen fluoride',
+    formula: 'FH',
+    CAS: '7664-39-3',
+    molecularWeight: 20.0063432,
+    C1: 0.7516,
+    C2: -0.0010874,
+    minimumTemperature: 189.79,
+    maximumTemperature: 394.45,
+    thermalConductivityAtMinimumTemperature: 0.5452,
+    thermalConductivityAtMaximumTemperature: 0.3227,
+  },
+
+  HydrogenSulfide: {
+    name: 'Hydrogen sulfide',
+    formula: 'H2S',
+    CAS: '04-06-7783',
```

Comment:
```suggestion
    CAS: '7783-06-04',
```

Be more mindful of these mistakes.